pep-0581.rst

```diff
@@ -0,0 +1,362 @@
+figure out how to build the integrations with GitHub.
+
+By using GitHub issues, where the source code is currently hosted, where pull
+requests are taking place, we'll be providing consistent experience to contributors
+and maintainers, and not having to jump from one interface to another.
+
+If someone else wants to do the research of what it takes for Python to start
+using other issue tracker, and what the migration plan is, please go ahead and
+come back with a PEP.
+
+Why not focus on improving Roundup / bpo
+----------------------------------------
+
+GitHub has many features we like that are already available. We still need to
+build out additional integrations and update our bots, but this is something
+we already know how to do.
+
+In order to really improve Roundup / bpo, it needs to first migrate to GitHub,
+add CI and bots. As I understand it, there is hesitation because upstream Roundup
+is still im Mercurial. Someone more familiar with Roundup / bpo needs
+to champion this effort. (I'm not volunteering, I'm sorry).
+
+I believe the effort of creating and maintaining GitHub integrations and bots
+is much less than the effort needed to get Roundup up to speed and then to continue
+maintaining it.
+
+Migration Plan
+==============
+
+Backup of GitHub data
+---------------------
+
+This effort has been started and being tracked as an issue in core-workflow
+[8]_. We're using GitHub Migrations API [9]_
+to download GitHub data for CPython on daily basis. The archives will be
+dropped in an S3 bucket.
+
+Thanks to Ernest W. Durbin III for working on this.
+
+Update the CLA host
+-------------------
+
+At the moment the CLA is hosted within bpo. It needs to be updated such that
+signing the CLA does not require a bpo account, and it should be hosted outside
+of the bpo.
+
+The current CLA process itself is not ideal. Currently contributors to
+devguide, peps, and core-workflow need to sign a CLA,  and it requires a bpo
+account. A bpo account should not be required for those projects.
+
+Currently the CLA process involves real human to manually check the records.
+Question: Will it be possible to completely automate the CLA process, so
+it does not require human intervention?
+
+Create labels for issue triage
+------------------------------
+
+In bpo, we currently have the following fields for each issues:
+
+Type: behavior, crash, compile error, resource usage, security, performance, enhancement.
+Components: 2to3, Argument Clinic, asyncio, Build, Cross-build, ctypes, ...
+Priority: release blocker, deferred blocker, critical, high, normal, low
+
+We will create the corresponding labels::
+
+   type-behavior, type-crash, type-compile error, type-resource usage, ...
+
+   components-2to3, components-argument clinic, components-asyncio, ...
+
+   priority-release blocker, priority-deferred blocker, priority-critical, ...
+
+In addition, we'll create ``needs triage`` label.
```

Comment:
I think it could be helpful to utilize GitHub's multiple issue templates [(more info here)](https://blog.github.com/2018-05-02-issue-template-improvements/). I found this to be really helpful on other OSS projects to keep general usage bugs out of the issue tracker by redirecting them elsewhere. 

IMO, Babel [does this really well](https://github.com/babel/babel/issues/new/choose) (and has great emoji usage too).